Assets/GameBrowserEntry.cs

```diff
@@ -1,24 +1,41 @@
 ﻿using Fumbbl;
 using Fumbbl.Api.Dto.Match;
-using System.Collections;
 using UnityEngine;
-using UnityEngine.Networking;
 using UnityEngine.UI;
 
 public class GameBrowserEntry : MonoBehaviour
 {
-    public TMPro.TextMeshProUGUI team1;
+    private Current matchDetails;
+
+    public Image progressBar;
     public Image team1Image;
-    public TMPro.TextMeshProUGUI team1Score;
+    public Image team2Image;
+    public TMPro.TextMeshProUGUI team1;
     public TMPro.TextMeshProUGUI team1Info;
+    public TMPro.TextMeshProUGUI team1Score;
     public TMPro.TextMeshProUGUI team2;
-    public Image team2Image;
-    public TMPro.TextMeshProUGUI team2Score;
     public TMPro.TextMeshProUGUI team2Info;
-    public Image progressBar;
+    public TMPro.TextMeshProUGUI team2Score;
     public TMPro.TextMeshProUGUI turnIndicator;
 
-    private Current matchDetails;
+
+
+    ///////////////////////////////////////////////////////////////////////////
+    //  MONOBEHAVIOUR METHODS  ////////////////////////////////////////////////
+    ///////////////////////////////////////////////////////////////////////////
+
+    public void OnClick()
+    {
+        FFB.Instance.Stop();
+        FFB.Instance.Connect(matchDetails.id);
+        MainHandler.Instance.SetScene(MainHandler.SceneType.ConnectScene);
+    }
+
+
+
+    ///////////////////////////////////////////////////////////////////////////
+    //  CUSTOM METHODS  ///////////////////////////////////////////////////////
```

Comment:
Don't need to state something as generic as "custom methods". Use `#region` to group monobehaviour and leave these outside that region.